drivers/adc/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@
 zephyr_library()
 
 zephyr_library_sources_ifdef(CONFIG_ADC_SHELL		adc_shell.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_GECKO           adc_gecko.c)
```

Comment:
It would be better to use tabs instead of these spaces before `adc_gecko.c`, for consistency with other entries.

Reply:
Fixed